test/src/unit-lru_cache.cc

```diff
@@ -75,9 +75,9 @@ TEST_CASE_METHOD(LRUCacheFx, "Unit-test class LRUCache", "[lru_cache]") {
   CHECK(!success);
 
   // Prepare some vectors
-  auto v1 = new int[3];
-  auto v2 = new int[3];
-  auto v3 = new int[3];
+  auto v1 = static_cast<int*>(std::malloc(sizeof(int) * 3));
+  auto v2 = static_cast<int*>(std::malloc(sizeof(int) * 3));
+  auto v3 = static_cast<int*>(std::malloc(sizeof(int) * 3));
```

Comment:
Can't we just add 

```
delete [] v1;
delete [] v2;
delete [] v3;
delete [] v4;
```

at the end of the test? It seems to have been forgotten.

Reply:
these are freed by the LRU cache